pylops/signalprocessing/dct.py

```diff
@@ -0,0 +1,137 @@
+__all__ = ["DCT"]
+
+from typing import List, Optional, Union
+
+import numpy as np
+from scipy import fft
+
+from pylops import LinearOperator
+from pylops.utils._internal import _value_or_sized_to_tuple
+from pylops.utils.decorators import reshaped
+from pylops.utils.typing import DTypeLike, InputDimsLike, NDArray
+
+
+class DCT(LinearOperator):
+    r"""Discreet Cosine Transform
+
+    Performs discreet cosine transform on the given multi-dimensional
+    array along the given axis.
+
+    Parameters
+    ----------
+    dims : :obj:`list` or :obj:`int`
+        Number of samples for each dimension
+    type : :obj:`int`, optional
+        Type of the DCT (see Notes). Default type is 2.
+    axes : :obj:`list` or :obj:`int`, optional
+        Axes over which the DCT is computed. If not given, the last len(dims) axes are used,
+        or all axes if dims is also not specified.
+    dtype : :obj:`str`, optional
+        Type of elements in input array.
+    workers :obj:`int`, optional
+        Maximum number of workers to use for parallel computation. If negative, the value wraps around from os.cpu_count().
+    name : :obj:`str`, optional
+        Name of operator (to be used by :func:`pylops.utils.describe.describe`)
+
+    Attributes
+    ----------
+    shape : :obj:`tuple`
+        Operator shape
+    explicit : :obj:`bool`
+        Operator contains a matrix that can be solved explicitly
+        (``True``) or not (``False``)
+
+    Notes
```

Comment:
This is not good! You just copied scipy documentation, this is not allowed.

Please rewrite this, it doesn't need to be so long... the key is to mention the need for normalization, that's all

Reply:
fixed in [289d51a](https://github.com/PyLops/pylops/pull/480/commits/289d51ad61c7294451d777af6367ea282ad1e78e)